ApiBundle/Exceptions/HttpException/AreaTransformerHttpException.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 namespace OpenOrchestra\ApiBundle\Exceptions\HttpException;
+use OpenOrchestra\BaseApi\Exceptions\ApiException;
```

Comment:
And this is not the right class to use.

The ApiException to extend is the one in the OpenOrchestra\BaseApi\Exceptions\HttpException namespace